src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -301,7 +301,6 @@ class WorkspaceMembersPage extends React.Component {
         const removableMembers = [];
         let data = [];
         _.each(policyMemberList, (policyMember, email) => {
-            if (policyMember.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) { return; }
             if (email !== this.props.session.email && email !== this.props.policy.owner) {
                 removableMembers.push(email);
```

Comment:
Removing `(policyMember.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)` will allow addition of those members to `removableMembers`. Will it not cause any problem?

How about adding `(policyMember.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)` in the above if statement?

Reply:
Hi @sobitneupane, I don't see any issue when removing removing `if (policyMember.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)` statement.

Reply:
Removing `(policyMember.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)` will allow addition of pending deleted members to `removableMembers`. I think crossed members deleted offline can be selected if you remove this line.

Reply:
@sobitneupane  Thanks for pointing out it. I am fixing and testing again locally

Reply:
oh, I saw the problem, I'll update my PR. Thanks @sobitneupane

Reply:
@sobitneupane I fixed your comment, please help to check the change code and steps to test.